components/gitpod-protocol/src/util/logging.ts

```diff
@@ -6,7 +6,7 @@
 
 const inspect: (object: any) => string = require("util").inspect; // undefined in frontend
 
-let plainLogging: boolean = false; // set to true during development to get non JSON output
+let plainLogging: boolean = true; // set to true during development to get non JSON output
```

Comment:
This should also be part of the "dev commit". Ideally, we should not need this, at all. 

:cloud: I think we had a de-jsonifier in the past. Could we use that together with `yarn telepresence` to avoid the need for this code change? Or maybe set an env var, or similar?

Reply:
Yes, I'll remove it as well, of course.